bench-tps/src/bench.rs

```diff
@@ -122,8 +126,17 @@ fn generate_chunked_transfers(
             &dest_keypair_chunks[chunk_index],
             threads,
             reclaim_lamports_back_to_source_account,
+            &mut old_blockhash,
```

Comment:
seems unused here

Reply:
This is needed to keep track of the most recent blockhash that's different from out current blockhash, in between calls into generate_txs, since we call generate_txs multiple times here, so we need to somehow "remember" it.